common/src/main/java/com/microsoft/identity/common/internal/result/MsalBrokerResultAdapter.java

```diff
@@ -628,4 +629,20 @@ public boolean getDeviceModeFromResultBundle(@NonNull final Bundle bundle) throw
     public @NonNull ClientException getExceptionForEmptyResultBundle() {
         return new ClientException(INVALID_BROKER_BUNDLE, "Broker Result not returned from Broker.");
     }
+
+    /**
+     * Deserializes the {@link GenerateShrResult} object from the broker response {@link Bundle}.
+     * @param resultBundle The result Bundle produced by the broker.
+     * @return The deserialized GenerateShrResult object containing the result (or corresponding
+     * error).
+     */
```

Comment:
seems to need formatting

Reply:
9ad1ad613d45e16b2cc0cba623a2e74998f0dcf7